docset/windows/storage/Disable-PhysicalDiskIdentification.md

```diff
@@ -52,11 +52,9 @@ This cmdlet requires a storage enclosure that supports SCSI Enclosure Services (
 
 ### Example 1: Disable the identification LED on all physical disks in a pool
 ```
-PS C:\>$StPool = (Get-StoragePool -FriendlyName "SpacePool")
-PS C:\> Disable-PhysicalDiskIndication -StoragePool $StPool
+PGet-PhysicalDisk -StoragePool $pool | Disable-PhysicalDiskIdentification
```

Comment:
This should be in the format of:

```
PS C:\> Get-PhysicalDisk -StoragePool $pool | Disable-PhysicalDiskIdentification
```

I'm assuming the "P" in front of "Get-" was a typo.

Where is $pool being initialized?